### PR TITLE
Add .rosinstall for fetch on kinetic

### PR DESCRIPTION
**Pull request by 708yamaguchi**
kineticのPCからfetchを動かすための環境を作れるように、kinetic用の.rosinstallを追加しました。

### CODE REVIEW

File: jsk_fetch_robot/jsk_fetch.rosinstall.kinetic [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    version: indigo
+- git:
+    local-name: ros/nodelet_core
+    uri: https://github.com/ros/nodelet_core
+    version: 1.9.6
+- git:
+    local-name: strands-project/mongodb_store
+    uri: https://github.com/strands-project/mongodb_store.git
+    version: kinetic-devel
+# when fetch_moveit_config 0.7.13 is released in apt, lines below can be removed.
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch.rosinstall.kinetic]: 0.7.13 is already released

File: jsk_fetch_robot/jsk_fetch.rosinstall.kinetic [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    uri: https://github.com/furushchev/audio_common.git
+    version: device
+- git:
+    local-name: ros-perception/image_pipeline
+    uri: https://github.com/ros-perception/image_pipeline
+    version: indigo
+- git:
+    local-name: ros/nodelet_core
+    uri: https://github.com/ros/nodelet_core
+    version: 1.9.6
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch.rosinstall.kinetic]: latest nodelet core is 1.9.14, is there any regression between 1.9.6 to 1.9.14

File: jsk_fetch_robot/jsk_fetch.rosinstall.kinetic [hunk 3]
```diff
@@ -0,0 +1,37 @@
+- git:
+    local-name: jsk-ros-pkg/jsk_recognition
+    uri: https://github.com/jsk-ros-pkg/jsk_recognition
+    version: 1.2.2
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch.rosinstall.kinetic]: latest jsk_recognitoin is 1.2.5, any trouble?
http://repositories.ros.org/status_page/ros_kinetic_default.html?q=jsk_recognition

File: jsk_fetch_robot/jsk_fetch.rosinstall.kinetic [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- git:
+    local-name: jsk-ros-pkg/jsk_pr2eus
+    uri: https://github.com/jsk-ros-pkg/jsk_pr2eus.git
+    version: 0.3.7
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch.rosinstall.kinetic]: latest version is 0.3.10, why 0.3.7
http://repositories.ros.org/status_page/ros_kinetic_default.html?q=jsk_pr2eus

File: jsk_fetch_robot/jsk_fetch.rosinstall.kinetic [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- git:
+    local-name: FA-I-sensor
+    uri: https://github.com/RoboticMaterials/FA-I-sensor.git
+- git:
+    local-name: ros-drivers/audio_common
+    uri: https://github.com/furushchev/audio_common.git
+    version: device
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch.rosinstall.kinetic]: which launch file require device info?

File: jsk_fetch_robot/jsk_fetch.rosinstall.kinetic [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+- git:
+    local-name: ros-perception/image_pipeline
+    uri: https://github.com/ros-perception/image_pipeline
+    version: indigo
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch.rosinstall.kinetic]: kinetic release using indigo branch https://github.com/ros-gbp/image_pipeline-release/blob/master/tracks.yaml#L105

File: jsk_fetch_robot/jsk_fetch.rosinstall.kinetic [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+- git:
+    local-name: ros/nodelet_core
+    uri: https://github.com/ros/nodelet_core
+    version: 1.9.6
+- git:
+    local-name: strands-project/mongodb_store
+    uri: https://github.com/strands-project/mongodb_store.git
+    version: kinetic-devel
```
Comment (k-okada) [inline jsk_fetch_robot/jsk_fetch.rosinstall.kinetic]: if we need latest code than version 0.3.7, please add comment on "we're waiting for 0.3.8" and link to the issue URL for requesting release.
https://github.com/strands-project/mongodb_store/commits/kinetic-devel

**Comment by 708yamaguchi** [comment]
@wkentaro 
修正してくださってありがとうございます。

@k-okada 
https://github.com/jsk-ros-pkg/jsk_robot/pull/251
で追加されている、
```
NOT_TEST_INSTALL=true EXTRA_DEB="ros-indigo-convex-decomposition ros-indigo-ivcon
```
の役割がわからないので教えていただけないでしょうか。

**Comment by k-okada** [comment]
moved to #947

**Pull request closed without merging**